lib/mayaUsd/render/vp2RenderDelegate/draw_item.cpp

```diff
@@ -25,21 +25,20 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 Data holder for its corresponding render item to facilitate parallelized evaluation.
 */
-HdVP2DrawItem::HdVP2DrawItem(HdVP2RenderDelegate* delegate, const HdRprimSharedData* sharedData)
+HdVP2DrawItem::HdVP2DrawItem(
+    HdVP2RenderDelegate*     delegate,
+    const HdRprimSharedData* sharedData)
     : HdDrawItem(sharedData)
     , _delegate(delegate)
 {
     // In the case of instancing, the ID of a proto has an attribute at the end,
-    // we keep this info in _renderItemName so if needed we can extract proto ID
```

Comment:
A draw item now has multiple render items, so this member was renamed to _drawItemName